guides/release/pages.yml

```diff
@@ -249,6 +249,46 @@
   pages:
     - title: "Managing Dependencies"
       url: "index"
+- title: "Using TypeScript"
+  url: "using-typescript"
```

Comment:
one thing that is important once we release something is the URLs, we need to always keep redirects and never break a link. This means that we need to be sure that we are happy with the URLs that we are generating.

Personally I think `using` in this URL is too much 🤔 i.e. `/release/using-typescript/` is not as good as `/release/typescript/`. What do you think?